libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -1831,9 +1829,16 @@ void IRGeneratorForStatements::endVisit(MemberAccess const& _memberAccess)
 					// the call will do the resolving
 					break;
 				case FunctionType::Kind::DelegateCall:
-					define(IRVariable(_memberAccess).part("address"), _memberAccess.expression());
+				{
+					ContractType const& contractType = dynamic_cast<ContractType const&>(actualType);
+					if (contractType.contractDefinition().isLibrary(), "")
+						define(IRVariable(_memberAccess).part("address")) << linkerSymbol(contractType) << "\n";
+					else
+						define(IRVariable(_memberAccess).part("address"), _memberAccess.expression());
```

Comment:
How can this be reachable if it is a delegatecall?

Reply:
Right. It's not reachable. Removed.